### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by JannisHoch** [pypi/support#3851]
### PyPI Username

`JannisHoch`

### Reason for request

Lost access to email address as employer changed.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @JannisHoch,

Please email support@pypi.org and let us know what email address you would like associated with your account and we will proceed from there. Thanks.

**Comment by Thespi-Brain** [comment]
Hi @JannisHoch,

Please also reference this link https://github.com/pypi/support/issues/3851 in your email. Thanks.

**Comment by Thespi-Brain** [comment]
Hi @JannisHoch,

I am following up regarding your request. Are you still having an issue regarding your email address and logging in? Let us know! I will keep this issue open for another week unless I hear from you earlier. After that, if you are still having trouble, feel free to create a new request. Thanks!

**Comment by JannisHoch** [comment]
Yes, still struggling since I need to active my primary mail address but do not have access to this mail address anymore. Ideally, I'd change the primary mail but that's not possible with the currently set being verified - which is not possible unfortunately.

**Comment by Thespi-Brain** [comment]
Hi @JannisHoch,

Thanks for sending over your preferred email. I have initiated an account recovery process and instructions are being sent to your preferred email address. Please follow the instructions in that email and reply to it, and we will go from there. Thanks!

**Comment by Thespi-Brain** [comment]
Hi @JannisHoch,

Thanks for completing the verification process. The verification was successful! I have updated the primary email address on the account. Your 2FA has been reset and a password reset has been issued. You should be able to regain access to your account after completing the password reset. Let us know if you have any more issues. Thanks.
